.travis.yml

```diff
@@ -11,17 +11,26 @@ env:
 go:
     - 1.11.x
 
-install:
-  - make deps
-
-script:
-  - bash <(curl -s https://raw.githubusercontent.com/ipfs/ci-helpers/master/travis-ci/run-standard-tests.sh)
+install: true
+
+jobs:
+  include:
+    - stage: "build and test"
+      name: "gx"
+      script:
+        - make deps
+        - bash <(curl -s https://raw.githubusercontent.com/ipfs/ci-helpers/master/travis-ci/run-standard-tests.sh)
+
+    - name: "go mod"
+      script:
+        - export GO111MODULE=on
+        - make mod_deps
+        - bash <(curl -s https://raw.githubusercontent.com/ipfs/ci-helpers/master/travis-ci/run-standard-tests.sh)
 
 cache:
     directories:
         - $GOPATH/pkg/mod
+        - $GOPATH/src/gx
```

Comment:
You might also want `go env GOCACHE`.

Reply:
I don't think I can call a command there and there would be hardcoding a value as go doesn't populate the `$GOCACHE` env var itself.

Reply:
Literally `GOCACHE=/hardcode`. It's deterministic anyway. Not a blocker we can improve this later.